libpkg/pkgsign_ecc.c

```diff
@@ -813,6 +813,18 @@ _generate_private_key(struct ecc_sign_ctx *keyinfo)
 	return (EPKG_OK);
 }
 
+static inline void
+_specific_explicit_bzero(uint8_t *buf, size_t bufsz)
+{
+
+#ifdef __APPLE__
+	memset_s(buf, bufsz, 0, bufsz);
+#else
+	explicit_bzero(buf, bufsz);
+#endif
+}
```

Comment:
We can use memset_s everywhere, pkg is already flagged C11

Reply:
AFAIR it's not very well supported, e.g. FreeBSD specifies that `#define __STDC_WANT_LIB_EXT1__ 1` should be used for that purposes.

Reply:
In fact, the whole `Annex K` is only marginally supported even if -std=c11 is specified.